### PR TITLE
Feature/4-pre-commit-hook

### PR DESCRIPTION
**Pull request by OlegMikus**
Git pre-commit hook that calls linting script

### CODE REVIEW

File: hooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# INIT WORKING DIR
+# ===================================================
+cd "$(dirname "${BASH_SOURCE[0]}")"
+FILE_DIR=$(pwd)
+cd ..
+CWD="$(pwd)"
+
+bash lint.sh
+
+cd "${CWD}"
```
Comment (ishaurko) [inline hooks/pre-commit:20]: Hook just has to run linter, not copy itself to .git repo each time. Lines 20-22 are for cleanup scripts when user just installs or reinstalls the project

File: hooks/pre-commit [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+# INIT WORKING DIR
+# ===================================================
+cd "$(dirname "${BASH_SOURCE[0]}")"
+FILE_DIR=$(pwd)
+cd ..
+CWD="$(pwd)"
+
+bash lint.sh
```
Comment (ishaurko) [inline hooks/pre-commit:18]: use ./ to run executables

**Review by ishaurko — CHANGES_REQUESTED** [review]

File: main.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
-from flask import Blueprint, render_template
-from . import db
+from flask import Blueprint, jsonify, render_template
```
Comment (ishaurko) [inline main.py:1]: rm this file from PR

**Review by ishaurko — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (OlegMikus) [reply main.py:1]: Ok

Comment (OlegMikus) [reply main.py]: Done

**Pull request merged**